updatebreaches.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const DEBUG = !process.env.PUSH_TO_KINTO;
+
+const got = require("got");
+
+const KINTO_UPDATE_ENDPOINT = "https://settings-writer.prod.mozaws.net/v1/buckets/main-workspace/collections/fxmonitor-breaches/records";
+const PROD_RECORDS_ENDPOINT = "https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/fxmonitor-breaches/records";
+const HIBP_BREACHES_ENDPOINT = "https://haveibeenpwned.com/api/v2/breaches";
+
+const USERNAME = process.env.KINTO_USERNAME;
+const PASSWORD = process.env.KINTO_PASSWORD;
+
+if (!DEBUG && (!USERNAME || !PASSWORD)) {
+  console.error("Please set credentials in the environment.");
+  process.exitCode = 1;
+  return;
+}
+
+const AUTH = Buffer.from(`${USERNAME}:${PASSWORD}`).toString("base64");
+
+async function run() {
+  const RemoteSettingsBreachesSet = new Set((await got(
```

Comment:
Minor nit, but should this be `remote_settings_breaches_set` to be consistent with the other variable names in this script?

Reply:
I use underscores and caps for literal consts, which this is not - do you think it feels inconsistent as a new reader of the file? 🤔

Reply:
OK, that makes sense. I think I was just getting confused between the distinction of `KINTO_UPDATE_ENDPOINT` (UPPER_CASE) vs `RemoteSettingsBreachesSet` (CamelCase) vs `hibp_breaches` (snake_case).

Reply:
Good point, CamelCase'd all the things.